.github/workflows/ci.yml

```diff
@@ -8,6 +8,71 @@ on:
 
 
 jobs:
+  lint:
+    name: Lint Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
+
+      - name: Install protobuf
+        run: sudo apt install -y protobuf-compiler
```

Comment:
Is it possible to install protobuf compiler in xline CI like this, which is somewhat different from how it is done in xline?

Reply:
Yes. The difference is one installs using a package manager, the other installs manually and the versions.